Add info about project purpose (#15)
NASA-IMPACT/pyarc2stac: README.md

# Pyarc2stac

> A reverse proxy that dataset metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) as [STAC](https://stacspec.org/), 

Reviewer: The reverse proxy is actually [arc2stac](https://github.com/NASA-IMPACT/arc2stac). This repos is more for creating STAC collection using ArcGIS metadata. @amarouane-ABDELHAK can you confirm if this is correct?

Reviewer 2: Yes [arc2stac](https://github.com/NASA-IMPACT/arc2stac) is the reverse proxy

Author: Right! Accurate now?

> A reverse proxy that dataset metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) as [STAC](https://stacspec.org/), 
> so they can become discoverable alongside other STAC-indexed resources.

Author: ```suggestion
Extract metadata from ESRI data services (ImageServer, MapServer, FeatureServer, SomethingServer) and convert it to [STAC](https://stacspec.org/), 
to make these services discoverable alongside other STAC-indexed resources.
```
